### PR TITLE
Add NL locale

### PR DESCRIPTION
**Pull request by vdboor**
As you mentioned in #17, this is a new pull request with NL translations.

### CODE REVIEW

File: vies/locale/nl/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+#: __init__.py:111
+#, python-format
+msgid "%s is not a valid ISO_3166-1 country code."
+msgstr "%s is geen geldige ISO_3166-1 landcode."
+
+#: __init__.py:114
+#, python-format
+msgid "%s is not a VIES member country."
+msgstr "%s is niet een VIES lidstaat."
```
Comment (jieter) [inline vies/locale/nl/LC_MESSAGES/django.po]: "%s is geen VIES-lidstaat", 

zie ook http://ans.ruhosting.nl/e-ans/29/04/02/body.html en http://www.spatiegebruik.nl/regels.html

File: vies/locale/nl/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+#: fields.py:20
+msgid "Not a valid European VAT number."
+msgstr "Dit is geen geldig Europese BTW nummer"
```
Comment (jieter) [inline vies/locale/nl/LC_MESSAGES/django.po]: "Dit is geen geldig Europees BTW-nummer"

File: vies/locale/nl/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+#: fields.py:21
+msgid "VIES check VAT service currently unavailable."
+msgstr "VIES BTW-nummer verifiatie is momenteel onbeschikbaar."
+
+#: models.py:12
+msgid "A VIES VAT field."
+msgstr "Een VIES BTW nummer."
```
Comment (jieter) [inline vies/locale/nl/LC_MESSAGES/django.po]: "Een VIES BTW-nummer."

File: vies/locale/nl/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+msgid "%s is not a VIES member country."
+msgstr "%s is niet een VIES lidstaat."
+
+#: fields.py:20
+msgid "Not a valid European VAT number."
+msgstr "Dit is geen geldig Europese BTW nummer"
+
+#: fields.py:21
+msgid "VIES check VAT service currently unavailable."
+msgstr "VIES BTW-nummer verifiatie is momenteel onbeschikbaar."
```
Comment (jieter) [inline vies/locale/nl/LC_MESSAGES/django.po]: "VIES BTW-nummerverificatie is momenteel niet beschikbaar."

File: vies/locale/nl/LC_MESSAGES/django.po (hunk 3 above)
Comment (jieter) [inline vies/locale/nl/LC_MESSAGES/django.po]: Hmm, it also needs to reflect 'field' in english.

**Comment by vdboor** [comment]
@jieter agreed on all proposals you made. That's a much better translation!

**Comment by jieter** [comment]
:+1:

**Comment by codingjoe** [comment]
released in 2.2.1

**Comment by vdboor** [comment]
Thanks! :D

**Pull request closed without merging**